.github/renovate.json5

```
@@ -10,17 +10,17 @@
   "enabledManagers": ["custom.regex"],
   "customManagers": [
     {
-      // Matches AGP versions annotated with a "renovate: AGP version" comment
+      // Matches AGP versions in versions.json
```

```suggestion
      // Matches AGP versions in versions.json and gradle.properties
```

---

Good catch. Better yet, I'll just remove filenames from the comment. Might've made sense when there were 2 managers, but now it's just duplicating the `fileMatch` list.

```suggestion
      // Matches AGP versions in files with custom patterns
```